### ISSUE TITLE
VoiceOver: Mentions of users on other instances are not read in toots

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#362]
STR:
1. In version 1.10, the accessibility release, find a toot that mentions someone on another instance. For example, if you are viewing your home timeline on mastodon.social, find a toot that mentions someone on toot.cafe, chaos.social, or any other instance.
2. Turn on VoiceOver and have that toot read to you.
   * Expected: Toot should be read, including the mentioned user and their domain.
   * Actual: In place of that user name and instance name, only the text "link to  " is read.

Example: I am logged into toot.cafe. My discussion with @DagAgren contains toots from me to them. For example, [this toot](https://toot.cafe/@marcozehe/103090009143806160), when read back with VoiceOver, reads as:

```Marco Zehe (@marcozehe@toot dot cafe): @Link to  Oh, fantastic! Thanks for letting me know. :)
2 hours ago```

However, [their earlier reply](https://mastodon.social/@tootapp/103089948188747538) to me is read with the shortened mention of my Fediverse user name, e. g. with the at toot dot cafe bit stripped out.

I would expect that, given the above two toots, the experience of tootapp on mastodon.social will be the other way around, when viewed while logged into mastodon.social itself.

**Comment by DagAgren** [comment]
Ah, the stupid link detector thinks they are mailto: links... Fixed.
